src/components/views/dialogs/NewSessionReviewDialog.js

```diff
@@ -35,12 +38,20 @@ export default class NewSessionReviewDialog extends React.PureComponent {
         this.props.onFinished(false);
     }
 
-    onContinueClick = () => {
+    onContinueClick = async () => {
         const { userId, device } = this.props;
-        Modal.createTrackedDialog('New Session Verification', 'Starting dialog', DeviceVerifyDialog, {
+        const cli = MatrixClientPeg.get();
+        const request = await cli.requestVerification(
             userId,
-            device,
-        }, null, /* priority = */ false, /* static = */ true);
+            [verificationMethods.SAS, SHOW_QR_CODE_METHOD],
```

Comment:
I don't know the history, but I guess I'd expect all verif method to be available via a single object / module / enum, rather than spread out across multiple modules. Not a blocker, but seems possibly confusing.

Reply:
Yeah, good point. I guess we could remove the `method` parameter from `requestVerification` as clients other than Riot using the js-sdk can already specify which methods they want to support when creating the `MatrixClient` through the `verificationMethods` option. Will make note to change this in a subsequent PR.

Reply:
https://github.com/vector-im/riot-web/issues/12319